django/db/backends/sqlite3/schema.py

```diff
@@ -11,26 +11,20 @@
 class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
 
     sql_delete_table = "DROP TABLE %(table)s"
-    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s)"
+    sql_create_fk = None
+    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s) DEFERRABLE INITIALLY DEFERRED"
```

Comment:
Something along these lines would break on tables that haven't been rebuilt and still have `IMMEDIATE` foreign key constraints:

```python
with transaction.atomic():
    # The following line will immediately fail now that foreign key
    # constraints are enforced and were created as IMMEDIATE
    # instead of INITIALLY DEFERRED.
    Book.objects.create(author_id=1)
    Author.objects.create(id=1)
```

We might want to mention that constraints used to be built as `DEFERRABLE IMMEDIATE` in the _Foreign keys are now created with ``DEFERRABLE INITIALLY DEFERRED``_ section and might require a rebuild to allow the above pattern to be used.